packages/arb-token-bridge-ui/src/hooks/useIsTestnetMode.ts

```diff
@@ -1,12 +1,24 @@
 import { useLocalStorage } from '@uidotdev/usehooks'
+import { useEffect } from 'react'
+import { isNetwork } from '../util/networks'
+import { useNetworks } from './useNetworks'
 
 const testnetModeLocalStorageKey = 'arbitrum:bridge:settings:testnetMode'
 
 export const useIsTestnetMode = () => {
+  const [{ sourceChain }] = useNetworks()
```

Comment:
I don't think `useIsTestnetMode` should directly be coupled to `useNetworks`. We can pass in an initial value as a prop to `useIsTestnetMode` which we read from `useNetworks`.

Reply:
This also allows us to not change the API of `useIsTestnetMode`.

Reply:
moved it out to [useSyncQueryParamsToTestnetMode.ts](https://github.com/OffchainLabs/arbitrum-token-bridge/pull/1395/files#diff-be46ac6be18aadbb6a1ba56db7e1806ec576e41b84167629c65dc787e5030a82)